main.py

```diff
@@ -311,12 +309,13 @@ def status(message):
                 sensor.stop()
             break
         except Exception as e:
```

Comment:
On second thought, maybe we shouldn't catch this exception. We can talk more about this in person, but if there is an exception, I want to know about it. A good way to know if something is wrong is if a monitor stops sending data. How we currently have it, if there is an exception, a message will be logged and the data will be deleted. If it is an intermittent problem then we might not notice that data is being deleted. I think it would be better if the monitor just stopped working. We could then debug what is going on and address that particular exception.

What do you think?